content/es/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,164 @@
+---
+reviewers:
+- raelga
+title: ¿Qué es Kubernetes?
+content_template: templates/concept
+weight: 10
+card: 
+  name: concepts
+  weight: 10
+---
+
+{{% capture overview %}}
+Esta página ofrece una visión general sobre Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+Kubernetes es una plataforma portable y extensible de código abierto para
+administrar cargas de trabajo y servicios. Kubernetes facilita la automatización
+y la configuración declarativa. Tiene un ecosistema grande y en rápido crecimiento.
+El soporte, las herramientas y los servicios para Kubernetes están ampliamente disponibles.
+
+Google liberó el proyecto Kubernetes en el año 2014. Kubernetes se basa en [la experiencia de
+Google corriendo aplicaciones en producción a gran escala por década y media](https://research.google.com/pubs/pub43438.html), junto a las mejores ideas y prácticas de la comunidad.
+
+## ¿Por qué necesito Kubernetes y qué puede hacer por mi?
+
+Kubernetes tiene varias características. Puedes pensar en Kubernetes como:
+
+- una plataforma de contenedores
+- una plataforma de microservicios
+- una plataforma portable de nube
+
+y mucho más.
+
+Kubernetes ofrece un ambiente de administración **centrado en contenedores**. Kubernetes
+orquesta la infraestructura de cómputo, redes y almacenamiento para que las cargas de
+trabajo de los usuarios no tengan que hacerlo. Esto ofrece la simplicidad de las Plataformas
+como Servicio (PaaS) con la flexibilidad de la Infraestructura como Servicio (IaaS) y permite
+la portabilidad entre proveedores de infraestructura.
+
+## ¿Qué hace de Kubernetes una plataforma?
+
+A pesar de que Kubernetes ya ofrece muchas funcionalidades, siempre hay nuevos
+escenarios que se benefician de nuevas características. Los flujos de trabajo
+de las aplicaciones pueden optimizarse para acelerar el tiempo de desarrollo. La
+orquestación ad hoc, aceptable al principio de un proyecto, a menudo requiere una
+robusta automatización de gran escala. Es por ello que Kubernetes fue diseñada como
```

Comment:
```suggestion
robusta automatización cuando estás trabajando a gran escala. Es por ello que Kubernetes fue diseñada como
```

Reply:
El `at scale` se refiere a cuando has de aplicar orquestación con cargas de trabajo considerables, el `de gran escala` me suena un poco raro. No sé si tienes alguna idea mejor de como reparafrasearlo. El `at scale` / `planet scale` en software me parecen conceptos díficiles de escribir en castellano con la misma fuerza.

Qué opináis?

Reply:
La frase original era:

> Ad hoc orchestration that is acceptable initially often requires robust automation at scale.

Tienes razón, al releer me doy cuenta que la escala no aplica a la automatización, sino que cuando escalas, la orquestación ad hoc ya no es suficiente. Déjame pensar en una alternativa.

Reply:
Take a look at: https://github.com/kubernetes/website/pull/14585/commits/cb67ff51e6714754d7eb724d666a69f7b06b4fa0 which is now part of this PR.

Reply:
I just added a poll in the slack channel to work on figure out the best way of expressing the same thing: https://kubernetes.slack.com/archives/CH7GB2E3B/p1559369305020100

What I get from the original phrase is: 
>  _You can build your custom solution (ad hoc) that can solve and work perfectly in your local environment, but if you need to scale up your application, probably your custom solution will need a lot of work and automation to run at scale._ :thought_balloon:

Reply:
Fixed as per poll results. Thanks!